documentation/docs/developers/strategies/twap.md

```diff
@@ -0,0 +1,38 @@
+# TWAP
+
+## Overview
+
+TWAP strategy expands upon Simple Trade strategy. By the end of this part, you should be able to : 
+
+* Divide up an order into *n* individual orders
+* Add time delay between each individual orders
+
+## Config
+
+Modifications for the following fields have to be made to the `config_map` file :
```

Comment:
While it's useful information for the reader... the way this is worded feels like you're writing a report, again.

You're not writing the docs to tell the users that *you*'ve done some work. The user is not interested in what you've done. The user is interested in what's new and what they should pay attention to, in order to follow the TWAP code's logic. The contents for the two perspectives may be the same.. but the wording should be very different.